pages/linux/eject.md

```diff
@@ -0,0 +1,31 @@
+# eject
+
+> Eject cds, floppy disks and tape drives.
+
+- Display the default device:
+
+`eject -d`
+
+- Eject the default device:
+
+`eject`
```

Comment:
I would suggest swapping this item with the first one.

Reply:
When I was creating this, I actually wasn't sure which way round it should be.

I was going to put it first as it's the call without any arguments.

But, I then also thought, as people may not know what their default drive is, should the check be first in a logical order?